config/sync/core.extension.yml

```diff
@@ -41,6 +41,7 @@ module:
   dpl_fbi: 0
   dpl_fbs: 0
   dpl_fees: 0
+  dpl_filter_paragraphs: 0
```

Comment:
Should this be renenabled in `dpl_update`?

Reply:
Yes, and now it is :)